### ISSUE TITLE
Default image directory

### ISSUE THREAD

**Issue by philgoetz** [AppleCommander/AppleCommander#34]
I'm using Windows, and AppleCommander defaults to the Documents library.
I never willingly put anything into My Documents, for multiple reasons.
I have a directory for Apple 2 disk images, way down in my directory structure.
It would be nice to be able to set that as the default directory to look in, whether at compile time, or thru the GUI.

**Comment by a2geek** [comment]
First off, hopefully it remembered where you last loaded a disk image from? The current scheme is to just "remember" the location. If you're a developer, see [UserPreferences](https://github.com/AppleCommander/AppleCommander/blob/master/src/main/java/com/webcodepro/applecommander/ui/UserPreferences.java).

I *think* the behavior on first run is just your current directory. (At least, that's what I got on Linux once I removed the `~/.AppleCommander.preferences` file. I believe Java preferences under Windows do go into the Registry.)

For changing behavior, I'm not certain what the ramifications (or user experience) would be. Would a launch flag or an environment variable be useful to over-ride the settings? 

Another idea would maybe to have some "lock" capability to set those preferences? That "feels" more involved but likely a better user experience.

Thoughts or comments? Are you, by chance a developer who could take a crack at it? ;-)
